pystarport/cluster.py

```diff
@@ -924,7 +924,11 @@ def init_cluster(
     cmd=None,
     gen_compose_file=False,
 ):
-    config = expand_yaml(config_path, dotenv)
+    extension = config_path.rsplit('.', 1)[1]
```

Comment:
Use [.suffix](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) of the `Path` object.